composer.json

```diff
@@ -92,7 +92,9 @@
     "laminas/laminas-code": "^3.3.1",
     "laminas/laminas-paginator": "^2.7",
     "laminas/laminas-servicemanager": "^3.2",
-    "scheb/two-factor-bundle": "^3.26",
+    "scheb/2fa-bundle": "^5.3",
+    "scheb/2fa-google-authenticator": "^5.3",
+    "scheb/2fa-qr-code": "^5.3",
```

Comment:
@dvesh3 do we really need `scheb/2fa-qr-code`? As we're generating the QR code using `endroid/qr-code` here: https://github.com/pimcore/pimcore/blob/4fd1f8d1889b0c5e5f315b6aa1a6dad55fb2e362/bundles/AdminBundle/Controller/Admin/UserController.php#L821

Reply:
Good catch! reverted https://github.com/pimcore/pimcore/commit/69219ce19887a8f77746c59b7eaca8038ac7056f